diff -ruN arduino-1.6.1.orig/libraries/Bridge/src/Bridge.h arduino-1.6.1/libraries/Bridge/src/Bridge.h
--- arduino-1.6.1.orig/libraries/Bridge/src/Bridge.h	2015-03-10 14:19:56.000000000 +0100
+++ arduino-1.6.1/libraries/Bridge/src/Bridge.h	2015-03-14 14:43:28.461304458 +0100
@@ -19,6 +19,10 @@
 #ifndef BRIDGE_H_
 #define BRIDGE_H_
 
+#ifndef BRIDGE_BAUDRATE
+#define BRIDGE_BAUDRATE 250000
+#endif
+
 #include <Arduino.h>
 #include <Stream.h>
 
@@ -96,7 +100,7 @@
       // Empty
     }
 
-    void begin(unsigned long baudrate = 250000) {
+    void begin(unsigned long baudrate = BRIDGE_BAUDRATE) {
       serial.begin(baudrate);
       BridgeClass::begin();
     }
